README.md

```diff
@@ -261,9 +262,28 @@ there was an error, it will contain an error object.
 
 For more details about how cancelation works, please refer to our [dedicated documentation section](https://stuart.api-docs.io/v2/jobs/job-cancellation).
 
+### Validate a Job
+
+Before creating a Job you can validate it (control delivery area & address format). Asking for a pricing is **optional** and does not prevent you from creating a Job.
+
+```php
+$job = new \Stuart\Job();
+
+$job->addPickup('46 Boulevard Barbès, 75018 Paris');
+
+$job->addDropOff('156 rue de Charonne, 75011 Paris')
+    ->setPackageType('small');
+    
+$result = $client->validateJob($job);
+```
+
+The result will hold the boolean value `true` if the job is valid. If
+there was an error, it will contain an error object.
+
 ### Get a pricing
 
 Before creating a Job you can ask for a pricing. Asking for a pricing is **optional** and does not prevent you from creating a Job.
+This method don't validate the delivery area and may be validated by `validateJob($job)`
```

Comment:
I won't write this thing down, as it's unclear to me if it's a mechanism we want to provide, or a bug we have 😬  please remove this part, will take the action internally to validate this and update the API documentation if required. Thanks.